### PR TITLE
Update GenAI alarm descriptions with next steps

### PR DESCRIPTION
**Pull request by sanchitmalhotra126**
Update GenAI alarm descriptions with some more details and next steps that the SL on call can follow. Very much open to feedback on how to make these more useful or clearer!

Example on how this shows up in Cloudwatch
Screenshot 2024-12-06 at 11 00 55 AM

### CODE REVIEW

**Review by cat5inthecradle — APPROVED** [review]
Love the detailed formatting!

File: aws/cloudformation/standalone/gen_ai_curriculum/monitoring/alarms/alarmConfigurations.ts [hunk 1]
```diff
@@ -113,8 +119,13 @@ const browserIndices = BROWSERS.map((_, i) => i + 1);
 export const chatCompletionHighBrowserFailureRateConfiguration: PutMetricAlarmInput =
   {
     AlarmName: 'genai_chat_completion_high_browser_failure_rate',
-    AlarmDescription:
-      'Alarms when browser are experiencing a high failure rate attempting chat completion requests.',
+    AlarmDescription: `Browsers are experiencing a high failure rate attempting chat completion requests.
+
+*Next Steps:*
+- Check the [GenAICurriculum Dashboard](https://us-east-1.console.aws.amazon.com/cloudwatch/home?region=us-east-1#dashboards/dashboard/GenAICurriculum)
+- Check [browser logs](https://us-east-1.console.aws.amazon.com/cloudwatch/home?region=us-east-1#logsV2:log-groups/log-group/production-browser-events/log-events/production) in Cloudwatch
```
Comment (fisher-alice) [inline aws/cloudformation/standalone/gen_ai_curriculum/monitoring/alarms/alarmConfigurations.ts]: Nit: Is there a particular message (key words) we can recommend to search for? I  don't recall any aichat-specific messages being logged.

**Review by fisher-alice — APPROVED** [review]
Nice! And thanks for setting up the Slack integration for the new SNS topic!

**Comment by cnbrenci** [comment]
It might also be good to link to a bookmark in a google doc (maybe add a section to  https://docs.google.com/document/d/1T0Vwwg22isdgsf66mYiRtUZVKaSxHo1PW89HAnW_twk/edit?tab=t.0#heading=h.o4g1cmvuqgpp ) in case we want to add more instructions for quick mitigations or troubleshooting steps without having to update the alarm each time?

File: aws/cloudformation/standalone/gen_ai_curriculum/monitoring/alarms/alarmConfigurations.ts (hunk 1 above)
Comment (sanchitmalhotra126) [reply aws/cloudformation/standalone/gen_ai_curriculum/monitoring/alarms/alarmConfigurations.ts]: Good call, we can at least filter on `appName: 'aichat'` as a starting point. Will add.

**Comment by sanchitmalhotra126** [comment]
> It might also be good to link to a bookmark in a google doc (maybe add a section to https://docs.google.com/document/d/1T0Vwwg22isdgsf66mYiRtUZVKaSxHo1PW89HAnW_twk/edit?tab=t.0#heading=h.o4g1cmvuqgpp ) in case we want to add more instructions for quick mitigations or troubleshooting steps without having to update the alarm each time?

Oh good call! I'll add a link and section for this

**Pull request merged**